Makefile

```diff
@@ -12,14 +12,14 @@ ${PROG}:	${PROG}.c
 	@OSTYPE=`uname`; \
 	OS_CFLAGS=-D__OS_$${OSTYPE}__; \
 	if [ $$OSTYPE = "FreeBSD" ]; then \
-		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE ${HAS_FREEBSD_ACL}"; \
+		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE -DHAS_UTIMENSAT ${HAS_FREEBSD_ACL}"; \
```

Comment:
Put differently: this is a hamfisted assumption that's slightly better than the current assumption (utimensat is always available).

I would rather put my energy behind autoconf than fiddle around with the __FreeBSD_version checks (which are admittedly annoying to maintain/incorrect if people backport support and don't bump __FreeBSD_version).